package.json

```diff
@@ -29,6 +29,10 @@
     {
       "name": "Juri Leino",
       "email": "npm@line-o.de"
+    },
+    {
+      "name": "Duncan Paterson",
```

Comment:
Well, this is somewhat unrelated. Don't get me wrong I do not mind adding you to the list of maintainers.
Let's just assume this has security-implications ;)